File: commands/music/pauseResume.js
/* global controller guild */

module.exports = {
  name: 'pr',
  description: 'Pausar eller återupptar musiken.',
  aliases: [ 'pause', 'resume', 'pausa', 'fortsätt' ],
  group: 'music',
  usage: 'pr',
  example: 'pr',
  serverOnly: true,
  adminOnly: false,
  execute( message, args ) {
    if ( !guild.voiceConnection || !controller || controller.queue.length === 0 ) return message.reply( 'botten spelar inget.' ).then( msg => { msg.delete( 10000 ); } );
    controller.pauseResume();
  },
};

File: commands/music/pauseResume.go
package music

import (
	"github.com/digitalungdom-se/dub/internal"
	"github.com/digitalungdom-se/dub/pkg"
)

var PauseResume = internal.Command{
	Name:        "pr",
	Description: "Pausar eller återupptar musiken",
	Aliases:     []string{"pause", "resume", "pausa", "fortsätt"},
	Group:       "music",
	Usage:       "pr",
	Example:     "pr",
	ServerOnly:  true,
	AdminOnly:   false,

	Execute: func(ctx *pkg.Context, server *internal.Server) error {
		ctx.Delete()
		server.Controller.PauseResume()

		return nil
	},
}
